torch/utils/data/dataloader.py

```diff
@@ -55,7 +55,35 @@ class DataLoader(object):
             worker subprocess with the worker id (an int in ``[0, num_workers - 1]``) as
             input, after seeding and before data loading. (default: ``None``)
 
-    .. note:: By default, each worker will have its PyTorch seed set to
+    .. note:: When num_workers != 0, corresponding worker processes are created each time
+              iterator for the DataLoader is obtained (as in when you call
+              ``enumerate(dataloader,0)``).
+              At this point, the dataset, ``collate_fn`` and ``worker_init_fn`` are passed to each
+              worker, where they are used to access and initialize data based on the indices
+              queued up from the main process. This means that dataset access together with
+              its internal IO, transforms and collation runs in the worker, while any
+              shuffle randomization is done in the main process which guides loading by assigning
+              indices to load. Workers are shut down once the end of the iteration is reached.
+              
+              Since workers rely on Python multiprocessing, worker launch behavior is different
+              on Windows compared to Unix. While on Unix, fork() is used internally, allowing
```

Comment:
is `fork()` always used or it has to do with `multiprocessing.set_start_method`?

Reply:
@vadimkantorov made a good point. It depends on the start method. I think you should make it clear.

Reply:
I've updated the comments based on it.